src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +752,66 @@ const UsersConfigurationRow = ({
     );
 };
 
+const SshKeysRow = ({
+    id, item, onChange, idx, removeitem,
+}) => {
+    const [finalized, setFinalized] = useState(false);
+
+    const validateKey = (key) => {
+        cockpit.script(`echo ${key} | ssh-keygen -l -f /dev/stdin`, {})
+                .then(() => setFinalized(true))
+                .catch(() => setFinalized(false));
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230911-122710-2dcbda42-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)